### ISSUE TITLE
Scenario outlines don't do placeholder interpolation within tables

### ISSUE THREAD

**Issue by marnen** [meadsteve/white-bread#78]
If I have a scenario outline like this:
```gherkin
Scenario Outline: Interpolation
  Given I have a step with a table:
    | name   |
    | <name> |

  Examples:
    | name |
    | foo  |
```
Cucumber puts "foo" in the data table, as expected. White Bread seems to put "&lt;name&gt;". I think this is a bug, and I'll try to fix it soon.

**Comment by meadsteve** [comment]
Certainly looks like a bug to me. I haven't actually used Scenario outlines that much so there's definitely more scope for bugs. Again thanks for the 👀 on the project.

**Comment by marnen** [comment]
You're most welcome...and thanks for creating a usable Elixir implementation of Cucumber!

I do use scenario outlines more than many other Cucumber users do, because I like to use them to show that two values should be the same. In other words, I like to use them for this sort of thing:

```gherkin
Scenario Outline: User login
Given a user exists with e-mail address "<email>"
And I am logged in as "<email>"
When I go to the home page
Then I should see "Welcome, <email>!"

  Examples:
    | email           |
    | joe@example.com |
```

**Comment by marnen** [comment]
Fixed by #95.
